src/repo/secret/infisical.go

```diff
@@ -0,0 +1,38 @@
+package secret
+
+import (
+	"sync"
+
+	infisical "github.com/infisical/go-sdk"
+)
+
+type Secret struct {
+	client infisical.InfisicalClientInterface
+	mu     sync.Mutex
+}
+
+func NewSecret(secret *Secret) (*Secret, error) {
```

Comment:
now sure how to create Secret getting Secret as a param.
I think it's by mistake